src/sandbox/profile/request/attribute/attribute.js

```diff
@@ -0,0 +1,94 @@
+const Validation = require('../../../../yoti_common/validation');
+const { SandboxAnchor } = require('./anchor');
+
+/**
+ * @class SandboxAttribute
+ */
+class SandboxAttribute {
+  /**
+   * @param {string} name
+   * @param {string} value
+   * @param {string} derivation
+   * @param {boolean} optional
+   * @param {SandboxAnchor[]} anchors
+   */
+  constructor(name, value, derivation, optional, anchors) {
+    Validation.isString(name, 'name');
+    this.name = name;
+
+    Validation.isString(value, 'value');
+    this.value = value;
+
+    Validation.isBoolean(optional, 'optional');
+    this.optional = optional;
+
+    if (derivation !== null) {
+      Validation.isString(derivation, 'derivation');
+    }
+    this.derivation = derivation;
+
+    if (anchors !== null) {
+      Validation.isArrayOfType(anchors, SandboxAnchor, 'anchors');
+    }
+    this.anchors = anchors;
+  }
+
+  /**
+   * @returns {string}
+   */
+  getName() {
+    return this.name;
+  }
+
+  /**
+   * @returns {string}
+   */
+  getValue() {
+    return this.value;
+  }
+
+  /**
+   * @returns {*}
+   */
+  getDerivation() {
+    return this.derivation;
+  }
+
+  /**
+   * @returns {boolean}
+   */
+  getOptional() {
+    return this.optional;
+  }
+
+  /**
+   * @returns {SandboxAnchor[]}
+   */
+  getAnchors() {
```

Comment:
Does Node not implement the same helper methods as Python and Java to easily filter the sources and verifiers?  Don't think it's needed, but would be good to keep them all similar?

Reply:
_Java SDK_ has only `getAnchors()` - I think this is ok for sandbox testing purposes